web/react/components/signup_team_complete.jsx

```diff
@@ -164,7 +164,8 @@ TeamUrlPage = React.createClass({
         }
 
         var cleaned_name = utils.cleanUpUrlable(name);
-        if (cleaned_name != name) {
+        var urlRegex = /^[a-z0-9]+$/g;
```

Comment:
I thought it can have dashes in the middle?  is this the same as the backend method?